source/math/IntVectorList.ooc

```diff
@@ -0,0 +1,78 @@
+/*
+* Copyright (C) 2014 - Simon Mika <simon@mika.se>
+*
+* This sofware is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* This software is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this software. If not, see <http://www.gnu.org/licenses/>.
+*/
+use ooc-collections
+import math
+import FloatVectorList
+
+IntVectorList: class extends VectorList<Int> {
+	init: func ~default {
+		super()
+	}
+	init: func ~heap (capacity: Int) {
+		super(capacity)
+	}
+	init: func ~fromVectorList (other: VectorList<Int>) {
+		super(other _vector)
+		this _count = other count
+	}
+	copy: func -> This {
+		result := This new(this _count)
+		for (i in 0 .. this _count)
+			result add(this[i])
```

Comment:
Actually, this function already exists in `VectorList`. Do you even need to add it here?